### DISCUSSION TITLE
[A/L] (2021/22) Induzione - Esercizio 8

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#308] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152322142-9426723e-e774-4998-8cd2-1fffd9fdf5f8.png)

**Comment by Exyss** [comment]
![Esercizio 8](https://user-images.githubusercontent.com/65043207/152645757-93d934d5-f983-443c-9c07-d0069f2cc58f.jpg)
